functional-samples/tutorial.open-api-reference/sw-tips.js

```diff
@@ -0,0 +1,28 @@
+console.log('sw-tips.js');
```

Comment:
remove?

Reply:
I was using it to show where to find the logs for the service worker in the initial steps of the tutorial.